microapidev/comment-microapi
# Fixed issue #340:getSingleCommentVotes

**Before submitting your PR for review**

- Run `npm run lint` to find errors in code syntax/format
- Run `npm run lint:fix` to fix all auto-fixable errors in source code and auto-format with prettier
- Ensure you fix any linting errors displayed after running any of the above commands

**What does this PR do?**
Fixes #340 
Fixes #issue_number (replace issue_number with the issue number this PR solves)

Summarise the main tasks handled in the pull request

**description of Task to be completed?**
Write a test for getSingleCommentVotes controller
Describe (in detail) what the task completed in the pull request does as per the relevant task
Write a test for getSingleCommentVotes controller to make sure it works as expected.
**How should this be manually tested?**
`npm run test` if the controller is working as expected the test should pass
Give bullet point instructions on how to setup the code and test it manually (Assume no prior experience on JS or code)

**Any background context you want to provide?**

Any pertinent information that should be considered

**What is the link to the issue on Github?**

https://github.com/microapidev/comment-microapi/issues/340

**Questions:**

If something is unclear or you want some questions to be addressed by your peers, mention them here
